tests/TestCase.php

```diff
@@ -2,13 +2,15 @@
 
 namespace Meilisearch\Scout\Tests;
 
+use Laravel\Scout\ScoutServiceProvider;
 use Meilisearch\Scout\MeilisearchServiceProvider;
 
 abstract class TestCase extends \Orchestra\Testbench\TestCase
 {
     protected function getPackageProviders($app)
     {
         return [
+            ScoutServiceProvider::class,
```

Comment:
This makes sure that we can test the service provider and check if we correctly extended the EngineManager